llama.cpp

```diff
@@ -7501,6 +7523,18 @@ static void llama_set_inputs(llama_context & lctx, const llama_batch & batch) {
         }
     }
 
+    {
```

Comment:
This array can be quite big with large contexts, and is only used with alibi models.

```suggestion
    if (hparams.f_max_alibi_bias != 0.0f) {
```